### PR TITLE
IE11 View PDF Fix

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

File: web-client/src/views/CaseDetailInternal.jsx [hunk 1]
```diff
@@ -120,19 +119,17 @@ export default connect(
                       </td>
                       <td>
                         <span className="responsive-label">Title</span>
-                        <button
-                          className="pdf-link"
+                        <a
+                          href={`${baseUrl}/documents/${
```
Comment (kkoskelin) [inline web-client/src/views/CaseDetailInternal.jsx:123]: I'm guessing that this is a place where we may still wish to compose this URL within a computed, or perhaps a sequence?

File: web-client/src/views/CaseDetailPetitioner.jsx [hunk 2]
```diff
@@ -177,40 +176,40 @@ export default connect(
                   </tr>
                 </thead>
                 <tbody>
-                  {caseDetail.documents.map((item, idx) => (
+                  {caseDetail.documents.map((document, idx) => (
```
Comment (kkoskelin) [inline web-client/src/views/CaseDetailPetitioner.jsx:179]: 👊

**Review by kkoskelin — APPROVED** [review]
This is a good first-pass.  Though I'm under the impression we'll want to, if possible, move some of the url string-concatenation out of the components.

File: web-client/src/views/CaseDetailInternal.jsx (hunk 1 above)
Comment (ericsorenson) [reply web-client/src/views/CaseDetailInternal.jsx:123]: Good point

File: web-client/src/views/CaseDetailPetitioner.jsx [hunk 3]
```diff
@@ -177,40 +176,40 @@ export default connect(
...
                     <tr key={idx}>
                       <td className="responsive-title">
                         <span className="responsive-label">Activity date</span>
-                        {moment(item.createdAt).format('l')}
+                        {moment(document.createdAt).format('l')}
```
Comment (dknesek) [inline web-client/src/views/CaseDetailPetitioner.jsx:183]: computed

File: web-client/src/views/CaseDetailRespondent.jsx [hunk 4]
```diff
@@ -111,19 +110,17 @@ export default connect(
                       </td>
                       <td>
                         <span className="responsive-label">Title</span>
-                        <button
-                          className="pdf-link"
+                        <a
+                          href={`${baseUrl}/documents/${
```
Comment (dknesek) [inline web-client/src/views/CaseDetailRespondent.jsx:114]: computed

File: web-client/src/views/DocumentDetail.jsx [hunk 5]
```diff
@@ -189,7 +166,9 @@ class DocumentDetail extends React.Component {
             <div className="usa-width-two-thirds">
               <iframe
                 title={`Document type: ${document.documentType}`}
```
Comment (dknesek) [inline web-client/src/views/DocumentDetail.jsx:168]: computed

File: web-client/src/views/DocumentDetail.jsx [hunk 6]
```diff
@@ -189,7 +166,9 @@ class DocumentDetail extends React.Component {
...
-                src={this.state.documentUrl}
+                src={`${baseUrl}/documents/${
```
Comment (dknesek) [inline web-client/src/views/DocumentDetail.jsx:169]: computed

File: web-client/src/views/CaseDetailPetitioner.jsx [hunk 7]
```diff
@@ -177,40 +176,40 @@ export default connect(
...
                       <td>
                         <span className="responsive-label">Status</span>
-                        {item.isStatusServed && (
+                        {document.isStatusServed && (
                           <span>
                             R served on {moment(caseDetail.irsDate).format('L')}
                           </span>
                         )}
-                        {!caseDetail.irsSendDate && <span>{item.status}</span>}
+                        {!caseDetail.irsSendDate && (
```
Comment (dknesek) [inline web-client/src/views/CaseDetailPetitioner.jsx:210]: computed "show*" or "*visible"

File: web-client/src/views/CaseDetailPetitioner.jsx [hunk 8]
```diff
@@ -177,40 +176,40 @@ export default connect(
...
                         <span className="responsive-label">Activity date</span>
-                        {moment(item.createdAt).format('l')}
+                        {moment(document.createdAt).format('l')}
                       </td>
                       <td>
                         <span className="responsive-label">Title</span>
-                        <button
-                          className="pdf-link"
+                        <a
+                          href={`${baseUrl}/documents/${
```
Comment (dknesek) [inline web-client/src/views/CaseDetailPetitioner.jsx:188]: computed

File: web-client/src/views/CaseDetailPetitioner.jsx [hunk 9]
```diff
@@ -177,40 +176,40 @@ export default connect(
...
                       </td>
                       <td>
                         <span className="responsive-label">Filed by</span>
-                        {item.filedBy}
+                        {document.filedBy}
                       </td>
                       <td>
                         <span className="responsive-label">Status</span>
-                        {item.isStatusServed && (
+                        {document.isStatusServed && (
```
Comment (dknesek) [inline web-client/src/views/CaseDetailPetitioner.jsx:205]: computed "show*" or "*visible"

**Review by dknesek — COMMENTED** [review]
Our use of computeds seems to be spotty overall. How do we make this better? Should we add a DoD checklist item for this and other CA conventions we are trying to habitualize?

**Pull request merged**